tests/unit/pywbemcli/simple_disablepull_mock_script.py

```diff
@@ -64,23 +56,35 @@ def _setup(conn, server, verbose):
         # of the current script when it is executed using exec(), so we hard
         # code the file path. This requires that the tests are run from the
         # repo main directory.
-        this_file_path = 'tests/unit/pywbemcli/simple_interop_mock_script.py'
+        this_file_path = \
+            'tests/unit/pywbemcli/simple_disablepull_mock_script.py'
         assert os.path.exists(this_file_path)
 
-    mof_file = 'simple_interop_mock_model.mof'
-    dependent_files = [mof_file,
-                       'mock_interop.mof',
-                       'simple_mock_model.mof']
-    mof_path = os.path.join(os.path.dirname(this_file_path), mof_file)
-    conn.compile_mof_file(mof_path, namespace=None)
+    # Prepare an Interop namespace and namespace provider
 
-    # Disable the pull operations for this test
-    conn.disable_pull_operations = True
+    interop_mof_file = 'mock_interop.mof'
```

Comment:
Should we consider packaging the functionality for creating and populating the interope namespace into a single method in pywbemcli since we are trying to get users to use this as a standard way to create an interop namespace.

Reply:
It is too bad that we have multiple steps to the registration, start with file name use the register_dependent to convert from name to path and then pass that on to the pywbem dependent registration.

Reply:
I created issue #1121 to cover the idea of a single method for creating the interop namespace.